org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeCoordinatorHandler.java

```diff
@@ -373,9 +373,15 @@ private void initialiseZigBee() {
         networkManager.addNetworkStateListener(this);
         networkManager.addNetworkNodeListener(this);
 
+        int meshUpdateTime = MESH_UPDATE_TIME;
+        if (getConfig().get(CONFIGURATION_MESHUPDATEPERIOD) != null) {
+            logger.debug("Mesh Update Period {}", getConfig().get(CONFIGURATION_MESHUPDATEPERIOD));
+            meshUpdateTime = ((BigDecimal) getConfig().get(CONFIGURATION_MESHUPDATEPERIOD)).intValue();
+        }
+
         // Add the extensions to the network
         ZigBeeDiscoveryExtension discoveryExtension = new ZigBeeDiscoveryExtension();
-        discoveryExtension.setUpdatePeriod(MESH_UPDATE_TIME);
+        discoveryExtension.setUpdatePeriod(meshUpdateTime);
```

Comment:
The configuration is named `mesh update period`, whereas the variable storing the config value here is named `mesh update time`, which seems like a (minor) inconsistency. If you consider this sensible as well, you could adapt the name of the variable here to match.

Reply:
Yep - I agree - thanks.